cpu/ppc64_cpu_test.py

```diff
@@ -76,7 +80,8 @@ def test(self):
             process.system_output("ppc64_cpu --info")
             self.smt()
             self.core()
-            self.subcore()
+            if self.smt_subcores == 1:
```

Comment:
@sacsant Instead of having a flag, this can be directly checked.
ex:
if os.path.exists("/sys/devices/system/cpu/subcores_per_core"):
    self.subcore()